Classes/Security/Authentication/Provider/LdapProvider.php

```diff
@@ -102,7 +103,7 @@ public function authenticate(TokenInterface $authenticationToken)
             $authenticationToken->setAccount($account);
             $this->emitAccountAuthenticated($account, $ldapUser);
         } catch (\Exception $exception) {
-            $this->logger->log('Authentication failed: ' . $exception->getMessage(), LOG_ALERT);
+            $this->logger->log(LogLevel::ALERT, 'Authentication failed: ' . $exception->getMessage());
```

Comment:
This should be using `alert()` instead:
```suggestion
            $this->logger->alert('Authentication failed: ' . $exception->getMessage());
```
See https://github.com/neos/flow-development-collection/pull/1574 that was merged for Flow 6.0